src/test/java/com/google/api/codegen/testdata/python_docs_apis_no_path_templates.baseline

```diff
@@ -17,4 +17,6 @@ API types
 .. autosummary::
    :toctree:
 
+   example.__init__
```

Comment:
You know, I have never tried to build a Sphinx doc with an explicit `__init__` like this. We should make sure it works properly. (It probably does.)